src/components/ranking-chart/ranking-chart-component.jsx

```diff
@@ -55,7 +55,7 @@ function RankingChart({
   const tableRef = useRef();
 
   const scrollTableVertically = (distance) => {
-    tableRef.current.scroll(0, distance);
+    tableRef.current.scrollTo({top: distance});
```

Comment:
The table is not scrolling to the selected country on reload now. Could you please check this?